applications/asset_tracker_v2/doc/asset_tracker_v2_description.rst

```diff
@@ -110,8 +110,10 @@ The real-time configurations supported by the application are listed in the foll
 +--------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+----------------+
 | Accelerometer threshold        | Accelerometer threshold in m/s². Minimal absolute value in m/s² for accelerometer readings to be considered valid movement.          | 10 m/s²        |
 +--------------------------------+--------------------------------------------------------------------------------------------------------------------------------------+----------------+
-| No Data List (NOD)             | A list containing :ref:`data types <app_data_types>` that are not sampled by the application. Used to disable sampling from          | No entries     |
-|                                | sensor sources. Currently, the application supports APP_DATA_GNSS and APP_DATA_NEIGHBOR_CELLS in the NOD list.                       | (Request all)  |
+| No Data List (NOD)             | A list of strings that references :ref:`data types <app_data_types>` that will not be sampled by the application.                    | No entries     |
```

Comment:
I dont think I fully understand this. If the application wont sample these sensors, then how does it get the data? interrupts/events?